fn main() -> Result<(), Box<dyn std::error::Error>> {
    serie::run()
}
